leaf-ui/rappid-extensions/AssignmentsTable.js

```diff
@@ -360,13 +370,20 @@ var IntentionRelationshipView = Backbone.View.extend({
 // TODO: Get linkSrc name and linkDest name instead of ids
 var LinkRelationshipView = Backbone.View.extend({
     model: LinkBBM,
+    tagName: 'tr',
+
+    initialize: function(options){
+        this.linkSrc= options.linkSrc,
+        this.linkDest= options.linkDest
+    },
 
     template: [
         '<script type="text/template" id="item-template">',
-        '<tr><td> <%= linkType %> </td><td> <%= linkSrcID %> </td>',
-        '<td> <%= linkDestID %> </td>',
-        '<td><input id="linkAbsRelation" type="number" name="sth" value= "<% if (linkAbsTime === -1) {%> "" <%} else { %> linkAbsTime <% } %>" ></td>',
-        '<td><button id="unassign-abs-rel-btn" > Unassign </button></td> </tr>',
+        '<td><%= linkType %> | <%= postType %></td>',
```

Comment:
What happens if link is of type NBT or NBT and does not have a post type?

Reply:
This is for evolving functions tho? Or that's what I thought? Is it for all types of links?

Reply:
Its for evolving functions but also it looks like the logic (historically at least) also allows for NBT and NBD - Maybe ask @amgrubb about use cases for that?

Reply:
Yes, NBT & NBD should show up in Absolute Relationship Assignment Table. Historically, NBT/NBD have had a post type of `null`, but it doesn't have to stay this way if it presents a problem. We can always have the post type be `NBP` or `NBX` if that makes it easier.
<img width="915" alt="Screen Shot 2021-07-02 at 9 56 14 AM" src="https://user-images.githubusercontent.com/6596937/124285258-c293d400-db1b-11eb-9f9c-6e3a3f7b6891.png">